### DISCUSSION TITLE
double click dont select right word

### DISCUSSION THREAD

**Discussion by Kurt-Shiwz** [helix-editor/helix#10128] [General]
I double click `exec` and real select
![image](https://github.com/helix-editor/helix/assets/7554917/abbef498-1ee2-487e-a9c0-9332b45f97f9)

There is conf in vscode like wordSeparators="`~!@#$%^&*()-=+[{]}\\|;:'\",.<>/?", will helix support it?

**Comment by woojiq** [comment]
Are you sure Helix supports mouse selection? If you have `mouse = false` in your configuration, then your terminal is responsible for selection. If you have `mouse = true` then you can only change the cursor position.
What terminal emulator do you use?

**Comment by Kurt-Shiwz** [comment]
> Are you sure Helix supports mouse selection? If you have `mouse = false` in your configuration, then your terminal is responsible for selection. If you have `mouse = true` then you can only change the cursor position. What terminal emulator do you use?

I already set `mouse = false`, double click i want to select `exec`
i use `https://github.com/wez/wezterm`

**Comment by woojiq** [comment]
> I already set `mouse = false`, double click i want to select `exec`

Then it's not helix problem because mouse is disabled, all mouse interactions are handled directly by wezterm. By the way, you need to turn off rendering of "whitespaces" in helix if you want to select "exec". Or add "whitespace" char to the terminal word boundary: https://wezfurlong.org/wezterm/config/lua/config/selection_word_boundary.html?h=select
